api/backend.proto

```diff
@@ -58,12 +58,14 @@ option (grpc.gateway.protoc_gen_swagger.options.openapiv2_swagger) = {
 message GrpcFunctionConfig {
   string host = 1;
   int32 port = 2;
+  bool useSecure = 3;
```

Comment:
Should this be the CA cert that the client should trust rather than say I'm turning on TLS.